.config/sample-config.json

```
@@ -43,6 +43,38 @@
             ],
             "thumbnailPath": "assets/thumbnail.png",
             "suggested": false
+        },
+        {
+            "id": "excel-add-in-chartapi-analyze-data",
```

There will be a lot of similar suggestions, but we should favor talking about "charts" instead of the "chart APIs".
```suggestion
            "id": "excel-add-in-chart-analyze-data",
```